example/lambdas/snsS3Test/index.js

```diff
@@ -14,7 +14,8 @@ async function handler(event) {
   const message = JSON.parse(messageString);
   return s3.putObject({
     Bucket: message.cumulus_meta.system_bucket,
-    Key: `${message.meta.stack}/test-output/${message.cumulus_meta.execution_name}.output`
+    Key: `${message.meta.stack}/test-output/${message.cumulus_meta.execution_name}.output`,
+    Body: JSON.stringify(event, null, 2)
```

Comment:
This change is unrelated to updating the granule files schema, but I found debugging failures of the indexer easier if the content of the event was stored to S3.